.github/workflows/release-build.yml

```diff
@@ -1,12 +1,16 @@
 name: Build release artifacts
 
 on:
-  # Handles a 'helics-release-build' dispatch event type
-  # The event payload should contain these fields:
-  # packages - a list of which packages to build (everything, archive, msvc, installer, sharedlib)
-  # commit (optional) - the commit-ish to do a release build with
-  repository_dispatch:
+  # Picking a workflow branch that is a tag, in combination with using the commit field is likely to fail for magic git clone/checkout reasons
   workflow_dispatch:
+    inputs:
+      packages:
+        description: 'List of packages to build (everything, archive, msvc, installer, sharedlib)'
+        required: true
+        default: 'everything'
+      commit:
+        description: 'Commit-ish (tag, branchname, sha) for code to build (uses scripts from the same branch as the workflow)'
+        default: ''
   schedule:
     - cron: '15 09 * * *' # Run in the early hours of the morning (UTC)
```

Comment:
do we do anything with these builds?

Reply:
Catching changes that break the automated release builds, hopefully while they are still in develop and before the pending release has been merged into main.